indexing-service/src/main/java/io/druid/indexing/worker/executor/ExecutorLifecycle.java

```diff
@@ -91,30 +111,6 @@ public void start()
       throw Throwables.propagate(e);
     }
 
-    // Spawn monitor thread to keep a watch on parent's stdin
```

Comment:
@gianm How would you feel about forcing a node restart if you wanted to make absolutely sure all peons were dead?